great_expectations/core/usage_statistics/package_dependencies.py

```diff
@@ -195,7 +195,6 @@ class GXDependencies:
         "scikit-learn",
         "shapely",
         "simple_icd_10",
-        "sklearn",
         "sympy",
```

Comment:
Just wanting to check : should `sklearn` here be replaced with `scikit-learn`?

Reply:
Yes, but scikit-learn is already in that list. Ditto for other comment.